rest-service/manager_rest/resource_manager.py

```diff
@@ -1393,12 +1428,13 @@ def _finalize_create_deployment(self, deployment: models.Deployment):
     @staticmethod
     def get_deployment_parents_from_labels(labels):
         parents = []
+        labels = labels or []
         for label, value in labels:
```

Comment:
You might to change the naming to:
`label` ->  `label_key` 
`value` -> `label_value`

Reply:
sure will change that